README.md

```diff
@@ -184,7 +184,7 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
 4. Plugin development
 
    You can refer to [plugin development guide](docs/en/latest/plugin-develop.md), and sample plugin `example-plugin`'s code implementation.
-   Reading [plugin concept](docs/en/latest/architecture-design/plugin.md) would help you learn more about the plugin.
+   Reading [plugin concept](docs/en/latest/architecture-design/apisix.md) would help you learn more about the plugin.
```

Comment:
```suggestion
   Reading [plugin concept](docs/en/latest/terminology/plugin.md) would help you learn more about the plugin.
```

Reply:
`plugin.md`  has moved to `terminology`.

Reply:
Could we add some link linter?
CC @navendu-pottekkat

Reply:
We can add link linter but since pages are across multiple repos, it would be difficult to do it statically.

Reply:
Can't we add linter in github workflows. 
May I know the meaning of availability of pages on multiple repos?
CC @navendu-pottekkat

Reply:
I did not realise that this was a change in README. For this case, a linter in GitHub workflows should work. There should be some action available that we can use.

For APISIX documentation, we have multiple repos and documentation for each of the components are in different repos. This is pulled in the apisix-website repo during a build. So, pages could be linked across repos and I don't think we can have static lint checks to catch them. We may need to build the website and test the links from the built website using some workflow.

Reply:
> I did not realise that this was a change in README. For this case, a linter in GitHub workflows should work. There should be some action available that we can use.
> 
> For APISIX documentation, we have multiple repos and documentation for each of the components are in different repos. This is pulled in the apisix-website repo during a build. So, pages could be linked across repos and I don't think we can have static lint checks to catch them. We may need to build the website and test the links from the built website using some workflow.

@navendu-pottekkat  We already have one, pls see https://github.com/apache/apisix-website/issues/1047.